packages/experimental/RadioGroup/src/Radio/useRadio.win32.ts

```diff
@@ -29,7 +29,7 @@ export const useRadio = (props: RadioProps): RadioInfo => {
     subtext,
     value,
     disabled,
-    labelPosition = radioGroupContext.layout === 'horizontal-stacked' ? 'below' : 'after',
+    labelPosition = 'after', // disables labelPosition in win32
```

Comment:
I think this should be a hardcoded const. If we take it from props someone can override it by passing a different value

Reply:
Fixed, thanks for the comment. We are shipping it only on win32 and android for now, but I decided to keep the horizontal layout code so we can easily re-enable it later if needed to align with Fluent. I changed the fix to force the layers to resolve the horizontal layouts to the default vertical, like you suggested.